### ISSUE TITLE
[Suggestion] Grouping Checks to result in faster detection

### ISSUE THREAD

**Issue by SkyLicks** [funkemunky/Iris-Issues#163]
**Suggestion**
I am suggesting to group hacks under certain categories such as "Combat," "Movement," "World" hacks.

**Why are you suggestion this?**
When hackers hack, they tend to trigger more then just one check. For example, a Fly hacker might trigger Flight (A), Flight (B), and Flight (D) all with in close proximity of each other. 

Or a combat hacker may trigger autoclicker and kill aura checks both at once.

Combining violations from multiple checks in the same category could theoretically result in catching hackers quicker and with less false positives.

**Comment by Strvm** [comment]
I don’t agree with this, I prefer to have multiple checks for the same hack. For example I know on my server that some checks false due to specific stuff on my server, so regrouping false checks with real check would just make catching cheaters more complicated. What I would find cool about regrouping checks is to regroup them in the checks GUI. At the moment each checks are represented by a piece of paper, I think it should be set up so for example a book called « Flight » and when you click on it it opens a new GUI with each Flight check and a back button.

**Comment by Cqban** [comment]
As Strummm said grouping the checks into categories would be a bad idea for catching cheats but doing it in the chat would only be better because some hacks flag a lot of checks and we cant delay them right now

**Comment by SkyLicks** [comment]
I think we should group all the flight checks together (flight a b c..), all KA checks together and etc instead of persueing my original idea of grouping categories of checks together. Plus, this would be made configurable wether you want to use groups or not.
